src/bundle/Resources/public/css/modules/content-view.css

```diff
@@ -1,22 +0,0 @@
-.ez-content-view {
```

Comment:
this of course completely breaks the content view page. The point of those PR is to replace `<ez-server-side-content class="ez-content-view">` by `<ez-content-view>`, in this CSS that means:

```css
ez-content-view {
    display: fex;
}

ez-content-view .ez-content-view-column {
    flex: 1 0;
}
```

and so on